src/utils.c

```diff
@@ -378,11 +378,11 @@ SEXP coerceUtf8IfNeeded(SEXP x) {
 #include <zlib.h>
 #endif
 SEXP dt_zlib_version() {
-  char out[51];
+  char out[71];
 #ifndef NOZLIB
-  snprintf(out, 50, "zlibVersion()==%s ZLIB_VERSION==%s", zlibVersion(), ZLIB_VERSION);
+  snprintf(out, 70, "zlibVersion()==%s ZLIB_VERSION==%s", zlibVersion(), ZLIB_VERSION);
```

Comment:
is this intended to be here? was this causing an error on your build?

Reply:
This particular 50->70 isn't strictly necessary, but the out buffer _was_ too small and I got a build warning about it. Do you prefer I restore this one to 50?

Reply:
I think it's fine to keep -- could you share with us the output, or perhaps make a note in the comments there that on some platforms it takes certain format so it's wider

Reply:
Sure. The warning was -
```
utils.c: In function ‘dt_zlib_version’:
utils.c:385:72: warning: ‘zlib header files were not found when data.table was compiled’ directive output truncated writing 61 bytes into a region of size 50 [-Wformat-truncation=]
   snprintf(out, 50, "zlib header files were not found when data.table was compiled");
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~
In file included from /usr/include/stdio.h:862:0,
                 from dt_stdio.h:30,
                 from data.table.h:1,
                 from utils.c:1:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:64:10: note: ‘__builtin_snprintf’ output 62 bytes into a destination of size 50
   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        __bos (__s), __fmt, __va_arg_pack ());
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
It is generated by the _next_ line, which is >50 chars on all platforms:
```
snprintf(out, 50, "zlib header files were not found when data.table was compiled");
```

Reply:
oh! indeed that buffer is too small. good catch.

Reply:
no need to add a comment even